python/pylibvw.cc

```diff
@@ -330,7 +330,7 @@ void unsetup_example(vw_ptr vwP, example_ptr ae)
     }
   }
 
-  uint32_t multiplier = all.wpp << all.reg.stride_shift;
+  uint32_t multiplier = all.wpp << all.stride_shift;
```

Comment:
When I had it as a variable in weight_parameters, I initialized the weights in parse_regressor as 
```all.weights = *(new weight_parameters(length, all.weights.stride_shift())); ```
which is very hacky.  This is because the stride_shift is set in parse_modules() whereas weights is initialized in parse_sources() (line 1360,1361 in parse_args.cc). So I could either propagate the stride_shift back up and pass it to parse_sources or I could just put it in the vw struct. I think we decided on doing the latter (for now) on our last Skype call.